### ISSUE TITLE
Generating dna sequence embeddings with evaluation mode or not

### ISSUE THREAD

**Issue by HelloWorldLTY** [MAGICS-LAB/DNABERT_2#129]
Hi, I notice that in the example code, the model embeddings generation do not contain model.eval() instruction, will this cause the model to have different embeddings under the same inputs? Thanks.

**Comment by Zhihan1996** [comment]
Hey, the `from_pretrained` function by default does `model.eval()` so it's fine.
